content/vendor/additional-objects/kots-application.md

```diff
@@ -0,0 +1,12 @@
+---
+date: 2019-11-27
+linktitle: "KOTS Application"
+title: KOTS Application
+weight: 20010
+---
+
+The [KOTS Application spec](/v1beta1/application/) allows for the customization of the initial experience (title, icon, release notes) and various operational and diagnostic features (port forwarding, statusInformers, and custom graphs).
```

Comment:
I’m not sure I agree with the grouping here of “initial experience” and “diagnostic features”.  Let’s just list the features that can be enabled by adding this document type.